riverpod_app/ios/Podfile

```
@@ -0,0 +1,43 @@
+  end
+  raise "FLUTTER_ROOT not found in #{generated_xcode_build_settings_path}. Try deleting Generated.xcconfig, then run flutter pub get"
+end
+
+require File.expand_path(File.join('packages', 'flutter_tools', 'bin', 'podhelper'), flutter_root)
+
+flutter_ios_podfile_setup
+
+target 'Runner' do
+  use_frameworks!
+
+  flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
```

RuboCop (1.73)

[convention] Use `__dir__` to get an absolute path to the current file's directory.

(Style/Dir)